packages/react-core/src/components/Select/Select.tsx

```diff
@@ -470,6 +475,16 @@ class Select extends React.Component<SelectProps & InjectedOuiaProps, SelectStat
       );
     }
 
+    const generateSelectedBadge = () => {
+      if (customBadgeText !== null) {
+        return customBadgeText;
+      }
+      if (Array.isArray(selections) && selections.length > 0) {
```

Comment:
You can use the `hasAnySelections` variable instead of `Array.isArray(selections) && selections.length > 0`.